.github/workflows/test.yml

```diff
@@ -123,8 +123,8 @@ jobs:
   # CLIENT (without types test)
   #
   client:
-    timeout-minutes: 35
-    runs-on: ${{ matrix.os }}
+    timeout-minutes: ${{ github.event_name == 'schedule' && 70 || 35 }}
+    runs-on: ${{ github.event_name == 'schedule' && contains(matrix.os, 'buildjet') && 'ubuntu-latest' || matrix.os }}
```

Comment:
This could deserve a comment now to explain what happens here.
Def will need a few minutes to parse in a few months...